specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.4-preview.1/secrets.json

```diff
@@ -934,14 +948,19 @@
           "x-ms-client-name": "secretAttributes",
           "$ref": "#/definitions/SecretAttributes",
           "description": "The secret management attributes."
+        },
+        "rotation_policy": {
```

Comment:
Why is the policy on both the secret bundle and the secret attributes? Seems it should only be on one, just as it is with Keys.